tests/integration/install/test_installation.py

```diff
@@ -392,14 +393,29 @@ def test_check_inventory_database_exists(ws, installation_ctx):
         installation_ctx.workspace_installer.configure()
 
 
-def test_compare_remote_local_install_versions(installation_ctx) -> None:
-    installation_ctx.workspace_installation.run()
+def is_on_release_branch(checkout_root: str) -> bool:
+    """Check if running on a release branch"""
+    release_branches = {"prepare/"}
+    try:
+        out = subprocess.run(
+            ["git", "rev-parse", "--abbrev-ref", "HEAD"], stdout=subprocess.PIPE, check=True, cwd=checkout_root
+        )
+        current_branch = out.stdout.decode("utf8")
+        return any(current_branch.startswith(release_branch) for release_branch in release_branches)
+    except subprocess.CalledProcessError:
+        return False
 
-    @retried(on=[NotFound], timeout=timedelta(minutes=2))
-    def wait_for_installation_to_finish():
-        installation_ctx.installation.load(WorkspaceConfig)
 
-    wait_for_installation_to_finish()
+def test_compare_remote_local_install_versions(installation_ctx) -> None:
+    """This test is known to fail on a branch with a major or minor version bump.
+
+    The databricks.labs.ucx.__about__.__version__ is bumped on the release branch BEFORE the git tag is bumped. The git
+    tag is bumped during release AFTER merging the release branch. Therefore, there this test is expected to fail on a
+    release branch that introduces a major or minor version bump.
```

Comment:
```suggestion
    The databricks.labs.ucx.__about__.__version__ is bumped on the release branch BEFORE the git tag is created. (The git
    tag is created during release AFTER merging the release branch.) Therefore, there this test is expected to fail on a
    release branch because the local `__version__` has been updated in anticipation of the tag that will shortly be created.
```